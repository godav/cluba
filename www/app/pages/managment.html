<show-during-resolve></show-during-resolve>
<md-toolbar ng-show="show"  ng-cloak>
    <div class="md-toolbar-tools gradient">
        <md-button class="md-icon-button" aria-label="Side Panel" ng-click="openSideNavPanel()">
            <md-tooltip>
                Side Panel
            </md-tooltip>
            <md-icon class="md-default-theme" class="material-icons">
                &#xE5D2;
            </md-icon>
        </md-button>

        <span flex="" class="flex"></span>

        <md-button class="md-icon-button" aria-label="back" ng-click="back()">
            <md-icon class="material-icons">
                <i class="material-icons">&#xE5C4;</i>
            </md-icon>
        </md-button>

    </div>

</md-toolbar>

<md-content flex layout-padding class="rtl" md-theme="altTheme" ng-show="show"  ng-cloak>
    <md-toolbar class="md-primary gradient _md _md-toolbar-transitions">
        <div class="md-toolbar-tools rtl">
            <h2 class="md-flex">המועדונים שלי</h2>
        </div>
    </md-toolbar>
    <md-list class="rtl" >
        <md-list-item ng-repeat="club in clubes" ng-hide="checkRole(club)" ng-click="editClub(club)">
            <md-icon class="md-default-theme margin-left-32" class="material-icons">
                &#xE544;
            </md-icon>
            <p class="rtl" >
                {{club.name}}
            </p>
            <div class="md-secondary">
                <img style='width: 60%' ng-src="{{club.imageLogo}}">
            </div>
        </md-list-item>
        <md-divider></md-divider>
    </md-list>


</md-content>

<div layout="column" layout-fill ui-view ng-show="!show"  ng-cloak></div>



<main-side-nav></main-side-nav>