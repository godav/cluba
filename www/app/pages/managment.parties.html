<md-toolbar style="color:white;" id="managment-toolbar" ng-class="{'bottom-shadow': boolChangeClass}"  ng-show="partiesShow">
    <div class="md-toolbar-tools gradient">
        <md-button class="md-icon-button" aria-label="Side Panel" ng-click="openSideNavPanel()">
            <md-tooltip>
                Side Panel
            </md-tooltip>
            <md-icon class="md-default-theme" class="material-icons">
                &#xE5D2;
            </md-icon>
        </md-button>

        <span flex="" class="flex"></span>

        <md-button class="md-icon-button" aria-label="back" ng-click="back()">
            <md-icon class="material-icons">
                <i class="material-icons">&#xE5C4;</i>
            </md-icon>
        </md-button>

    </div>

    <md-content>
        <div layout="row" class="club-sub-head" >
            <div flex="20">
                <span class="club-name" >תאריך</span>
            </div>
            <div flex="40">
                <span class="club-name" >שם הליין</span>
            </div>
            <div flex="15">
                <span class="event-panding" >נרשמו</span>
            </div>
            <div flex="15">
                <span class="event-approved">אושרו</span>
            </div>
            <div flex="10">
                <span class="club-reg-btn1">עריכה</span>
            </div>
        </div>
    </md-content>
</md-toolbar>

<div scroll-headline-managment id="managment-content" flex style="overflow: auto; position: relative !important;" ng-clock  ng-show="partiesShow">
    <div ng-repeat="event in events" >	
        <hr ng-if="$first" class="club-hr-no-top" >
        <hr ng-if="!$first" class="club-hr">
        <div layout="row" class="club-list">

            <div flex="20" layout="row" style="margin-top: 8px; margin-bottom: 8px;">
                <span class="club-name" >{{event.eDate| date:'dd/MM/yy'}}</span>
            </div>
            <div flex="40" layout="row" style="margin-top: 8px; margin-bottom: 8px;">
                <span class="club-name" >{{event.name}}</span>
            </div>
            <div flex="15">
                <md-button class="md-fab md-mini" ng-click="pending(event.$id)" style="margin: 0px !important; background-color: orange !important;" aria-label="Eat cake">
                    {{event.pending.all}}
                </md-button> 
                <!--<span class="club-name circle-padding" >{{event.price}}</span>-->
            </div>
            <div flex="15">
                <md-button class="md-fab md-mini" ng-click="approved(event.$id)" style="margin: 0px !important; background-color: green !important;" aria-label="Eat cake">
                    {{event.approved.all}}
                </md-button>
                <!--<span class="event-approved circle-approved ">{{event.minAge}} </span>-->
            </div>
            <div flex="10" layout="row">

                <md-icon class="material-icons" ng-click="edit(event.$id)">
                    <span class="club-reg-btn1"><i class="material-icons">&#xE3C9;</i></span>
                </md-icon>

            </div>          
        </div>
        <hr ng-if="$last" class="club-hr-no-bottom">
    </div>

</div>

<md-button class="md-fab  md-mini md-fab-bottom-right fixed-fab-right" ng-click="newEvent()" ng-show="partiesShow">
    <md-icon class="material-icons">
        <i class="material-icons">&#xE145;</i>
    </md-icon>
</md-button>

<footer  ng-show="partiesShow && role < 3">
    <md-toolbar style="background-color:whitesmoke;">
        <div layout="row" layout-align="center center" flex>
            <md-button class="md-primary" ng-click="goPermissions()">
                הרשאות
            </md-button>

            <md-button class="md-primary" ng-click="goProfile()">
                פרופיל
            </md-button>

            <md-button class="md-primary" ui-sref="managment.stats">
                סטטיסטיקות
            </md-button>

            <md-button class="md-primary font-bold">
                מסיבות והרשמות
            </md-button>
        </div>
    </md-toolbar>
</footer>

<div layout="column" id="subParties" layout-fill ui-view ng-show="!partiesShow"></div>