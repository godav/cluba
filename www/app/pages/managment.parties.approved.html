<show-during-resolve></show-during-resolve>
<md-toolbar style="color:white;" id="pending-toolbar" ng-class="{'bottom-shadow': boolChangeClass}"  ng-show="!partiesShow"  ng-cloak>
    <div class="md-toolbar-tools gradient">
        <md-button class="md-icon-button" aria-label="Side Panel" ng-click="openSideNavPanel()">
            <md-icon class="md-default-theme" class="material-icons">
                &#xE5D2;
            </md-icon>
        </md-button>

        <span flex="" class="flex"></span>

        <md-button class="md-icon-button" aria-label="back" ng-click="back()">
            <md-icon class="material-icons">
                <i class="material-icons">&#xE5C4;</i>
            </md-icon>
        </md-button>


    </div>

    <md-content>
        <div layout="row" class="club-sub-head" >
            <div flex="25">
                <span class="club-name" >תמונה</span>
            </div>
            <div flex="45">
                <span class="club-name" >שם</span>
            </div>
            <div flex="15">
                <span class="event-panding" >מין</span>
            </div>

            <div flex="15">
                <span class="club-reg-btn1">אישור</span>
            </div>
        </div>
    </md-content>
</md-toolbar>
<div  id="toast-content" style="position: relative;">   </div>
<div scroll-headline-pending id="pending-content" flex style="overflow: auto;" ng-clock  ng-show="!partiesShow">

    <div ng-repeat="user in users" >	
        <hr ng-if="$first" class="user-hr-no-top" >
        <hr ng-if="!$first" class="user-hr">
        <div layout="row" class="user-list" ng-class="user.group % 2 === 0 ? 'even-line' : 'odd-line'">

            <div flex="25" layout="row" >
                <img ng-src='{{user.picture}}' class="avatar" ng-click="showConfirm($event, user)"> 
            </div>
            <div flex="45" layout="row">
                <span class="user-name vertical-text" >{{user.name}}</span>
            </div>
            <div flex="15" layout="row">
                <span class="user-name vertical-text" ><i ng-if="user.gender === 'male'" class="fa fa-mars" style="font-size:24px"></i><i  ng-if="user.gender === 'female'" class="fa fa-venus" style="font-size:24px"></i></span>
            </div>
            <div flex="15" layout="row">
                <md-checkbox ng-disabled="user.entered" ng-model="user.entered" aria-label="Checkbox 1"  ng-change="changeState(user)"></md-checkbox>

            </div>          
        </div>
        <hr ng-if="$last" class="user-hr-no-bottom">


    </div>


    <md-fab-toolbar ng-show="!showSearchBar()"  ng-class="isOpen ? 'toolbar-open' : 'toolbar-close'" md-open="isOpen" md-direction="left" count="countt" id="fab-tool">
        <md-fab-trigger class="align-with-text" style="right: 35px;">
            <md-button aria-label="options" class="md-fab md-mini md-primary"> 
                <md-icon style="display: unset">
                    <!-- Put what ever youwant here -->
                    <i class="material-icons">&#xE5D2;</i>
                </md-icon>

            </md-button>
        </md-fab-trigger>

        <md-toolbar>
            <md-fab-actions class="md-toolbar-tools" style="justify-content: space-around;">
                <md-button aria-label="search" class="md-icon-button" ng-click="initiateSearch()">
                    <i class="fa fa-search fa-2x" aria-hidden="true"></i>
                </md-button>
                <md-button aria-label="fiter by all" class="md-icon-button" ng-click="filterAll()">
                    <i class="fa fa-venus-mars fa-2x" aria-hidden="true"></i>
                </md-button>                
                <md-button aria-label="gender pie" class="md-icon-button" ng-click="chart()">
                    <i class="fa fa-pie-chart fa-2x" aria-hidden="true"></i>
                </md-button>
            </md-fab-actions>
        </md-toolbar>
    </md-fab-toolbar>

    <!-- Search bar -->
    <div ng-show="showSearchBar()" class="md-toolbar-tools search-bar">
        <md-button class="md-icon-button" ng-click="endSearch()">
            <md-icon>close</md-icon>
        </md-button>

        <!-- Input Text -->
        <md-input-container flex>
            <!-- <label for="search">Search</label> -->
            <input aria-label="search" id="search-input" type="text" ng-model="search" focus-me="showSearchBar()" style="text-align: right;" ng-model-options="{debounce: 500}"/>
        </md-input-container>

        <md-button class="md-icon-button" ng-click="submit()">
            <md-icon>search</md-icon>
        </md-button>
    </div>

    <div ng-show="noData" class="no-data"><span>אין משתמשים תואמים</span></div>
    
    <modal id="pending-gender-graphs" attachTo="">
        <div class="modal">
            <div class="modal-body">
                <div style="border: 1px solid">
                    <nvd3 id="genderEnteredGraph" api="api" options="options" data="data" class="with-3d-shadow with-transitions" config="config" ></nvd3>

                </div>
    
                <button ng-click="closeModal('pending-gender-graphs');">סגור</button>
            </div>
        </div>
        <div class="modal-background"></div>
    </modal>  
</div>
